pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -4278,6 +4278,34 @@ void FullscreenUI::DrawPatchesOrCheatsSettingsPage(bool cheats)
 		{
 			ActiveButton("No cheats are available for this game.", false, false,
 				ImGuiFullscreen::LAYOUT_MENU_BUTTON_HEIGHT_NO_SUMMARY);
+
+			//	debug button for creating pnach file if none exists, also extends to copy path to clipboard for accesibility
+			if (MenuButton(ICON_FA_MAGIC " Create Default Pnach", "creates a default pnach file for debug purposes.", true))
+			{
+				FILE*		file{};
```

Comment:
This whole block should be removed. There's no reason it should be in the big picture UI.

Or, if anything, it should only be in the Qt UI, since nobody's going to be writing cheats exclusively using a gamepad.

One potential way I could see it working would be a menu item, e.g. "Tools -> Edit Patches" and "Edit Cheats", which fills the template, and opens the file using the default system editor.